stellarphot/notebooks/photometry/07-transit-fit-with-exotic.ipynb

```diff
@@ -133,8 +133,8 @@
    "outputs": [],
    "source": [
     "print('COPY AND PASTE THE LINE BELOW INTO AN EMPTY CELL TO RUN EXOTIC (include the exclamation point at the beginning):\\n\\n')\n",
-    "print(f'!exotic {\" \".join(exotic_arguments[whole_thing.planet_type.value])} {settings_name}')\n",
-    "print(\"\\n\\n\")"
+    "print(f'!exotic {\" \".join(exotic_arguments[settings.planet_type.value])} {settings_name}')\n",
```

Comment:
This was the only significant change I could see in this file, I don't see whole_thing even defined in the file

Reply:
Yeah, `whole_thing` was a vestige of an earlier iteration